app/views/dashboard/_new_hosts_widget.html.erb

```diff
@@ -0,0 +1,11 @@
+<% hosts = Host.authorized(:view_hosts).includes(:owner, :operatingsystem).order(:created_at => :desc).limit(8) %>
```

Comment:
does it also make sense to do something based on installed_at attribute as well?